src/proxyvisor.coffee

```diff
@@ -57,9 +60,13 @@ router.get '/v1/assets/:commit', (req, res) ->
 		return res.status(404).send('Not found') if !app
 		getAssetsPath(app.imageId)
 	.then (path) ->
-		getTarArchive(path)
-	.then (archivePath) ->
-		res.sendFile(archivePath)
+		dest = '/tmp/' + app.imageId + '.tar'
```

Comment:
Also, the imageId might have slashes on it, so this might try to store in directories that don't exist. Maybe use the app.commit as a filename instead of imageId?